<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>搜索</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>编号 / 手机号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pSearchLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_pSearchBtn">
       <property name="text">
        <string>  搜索   </string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_pTreeWidget">
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>受检单位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>采样日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>添加时间</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>1</string>
      </property>
      <property name="text">
       <string>18135100170</string>
      </property>
      <property name="text">
       <string>晋A03510</string>
      </property>
      <property name="text">
       <string>2015-07-20</string>
      </property>
      <property name="text">
       <string>2015-07-20 18:33:32</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_pPageLabel">
       <property name="text">
        <string>第1页 / 共1页</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pFirstPageBtn">
       <property name="text">
        <string>首页</string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/images/pagehead.png</normaloff>:/images/pagehead.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pPreviousPageBtn">
       <property name="text">
        <string>上一页</string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/images/pageup.png</normaloff>:/images/pageup.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pNextPageBtn">
       <property name="text">
        <string>下一页</string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/images/pagedown.png</normaloff>:/images/pagedown.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pLastPageBtn">
       <property name="text">
        <string>尾页 </string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/images/pagetail.png</normaloff>:/images/pagetail.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
